### PR TITLE
poweroff: update Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by spageektti — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

File: pages.es/linux/poweroff.md [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
+
+`poweroff --halt`
+
+- Reinicia el sistema (igual que `reboot`):
+
+`poweroff --reboot`
+
+- Apaga inmediatamente sin contactarse con el administrador del sistema:
+
+`poweroff --force --force`
```
Comment (spageektti) [inline pages.es/linux/poweroff.md:20]:
```suggestion
`poweroff --force`
```
Shouldn't it be like this?

File: pages.es/linux/poweroff.md [hunk 2]
```diff
@@ -1,8 +1,24 @@
...
+> Apaga el sistema.
 > Más información: <https://www.manned.org/poweroff>.
 
-- Apaga la máquina:
+- Apaga el sistema:
 
-`sudo poweroff`
+`poweroff`
+
+- Detiene el sistema (igual que `halt`):
```
Comment (tricantivu) [inline pages.es/linux/poweroff.md:10]:
```suggestion
- Detén el sistema (igual que `halt`):
```

File: pages.es/linux/poweroff.md [hunk 3]
```diff
@@ -1,8 +1,24 @@
...
+
+`poweroff --halt`
+
+- Reinicia el sistema (igual que `reboot`):
+
+`poweroff --reboot`
+
+- Apaga inmediatamente sin contactarse con el administrador del sistema:
```
Comment (tricantivu) [inline pages.es/linux/poweroff.md:18]:
```suggestion
- Apaga inmediatamente el sistema sin contactar al administrador:
```

File: pages.es/linux/poweroff.md [hunk 4]
```diff
@@ -1,8 +1,24 @@
...
+
+`poweroff --force`
+
+- Escribe la entrada wtmp sin apagar el sistema:
```
Comment (tricantivu) [inline pages.es/linux/poweroff.md:22]:
```suggestion
- Escribe una entrada en el archivo wtmp sin apagar el sistema:
```

**Pull request merged**